### PR TITLE
Fix shim301db build

### PR DESCRIPTION
**Pull request by gerashegalov**
Signed-off-by: Gera Shegalov <gera@apache.org>

#2172 broke the shim301db build

### CODE REVIEW

**Comment by gerashegalov** [comment]
build

File: shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/GpuBroadcastNestedLoopJoinExec.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ case class GpuBroadcastNestedLoopJoinExec(
     joinType: JoinType,
```
Comment (jlowe) [inline shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/GpuBroadcastNestedLoopJoinExec.scala:34]: 2021 copyrights

**Review by jlowe — DISMISSED** [review]

**Comment by jlowe** [comment]
build

**Comment by tgravescs** [comment]
the file rename here looks like a typo, remove it.

**Review by tgravescs — APPROVED** [review]

**Comment by gerashegalov** [comment]
build

**Pull request merged**